doc/source/whatsnew/v0.21.0.txt

```diff
@@ -179,6 +179,7 @@ Sparse
 ^^^^^^
 
 
+- Bug in instantiating :class:`SparseSeries` from ``dict`` with or without ``index=`` kwarg (:issue:`16905`)
```

Comment:
Fair enough, though ultimately instantiating from ``dict`` just didn't work at all though, which could be considered a single bug (also you're only referencing one issue here).  Note that without further context, people won't be aware of that difference, so it is preferable to be concise.